internal/datanode/data_node.go

```diff
@@ -374,19 +377,20 @@ func (node *DataNode) handleWatchInfo(e *event, key string, data []byte) {
 		log.Info("DataNode is handling watchInfo DELETE event", zap.String("key", key))
 	}
 
-	actualManager, loaded := node.eventManagerMap.LoadOrStore(e.vChanName, newChannelEventManager(
+	actualManager, loaded := node.eventManagerMap.GetOrInsert(e.vChanName, newChannelEventManager(
```

Comment:
the `newChannelEventManager` will miss to close if the key exists

Reply:
new channelEventManager here will not be started in key already exists